src/net/sf/memoranda/ui/ExportSticker.java

```diff
@@ -69,12 +69,12 @@ public boolean export(String src){
                         fwrite.write(contents);
                         
                         fwrite.close();
-                        JOptionPane.showMessageDialog(null,Local.getString("Documento creado con exito en su carpeta Memoranda =D"));
+/*TJ*/                        JOptionPane.showMessageDialog(null,Local.getString("Document created with success in its Memoranda folder =D"));
             
             
         } catch (IOException e) {
             e.printStackTrace();
-            JOptionPane.showMessageDialog(null,Local.getString("NO Logramos crear su documento =(..."));
+/*TJ*/            JOptionPane.showMessageDialog(null,Local.getString("We failed to create your document =(..."));
         }

```

Comment:
Language changes reviewed by ghgofort 12:23am - 2/8/16. All changes look ready to commit.